.circleci/config.yml

```diff
@@ -6,7 +6,7 @@ jobs:
       ignore:
         - gh-pages
     docker:
-      - image: python:3.7
+      - image: python:3.9
```

Comment:
This should probably match the version in our Dockerfile so that our CI and deployments are using the same Python version. I _think_ we're on 3.8?

Reply:
`FROM python:3.8-slim-buster as base`

ok, i'll try that!